justfile

```diff
@@ -33,6 +33,9 @@ test_async_std_pkg_all pkg=test_pkg:
 test_async_std_pkg_test name=test_name:
   cargo test --verbose --release --features=async-std-executor,demo,channel-async-std --lib --bins --tests --benches --workspace --no-fail-fast _nodes -- --test-threads=1 --nocapture
 
+list_tests_json:
+  cargo test --verbose --profile=release-lto --features=full-ci --lib --bins --tests --benches --package=hotshot-testing --no-fail-fast -- --test-threads=1 --nocapture -Zunstable-options --format json
+
```

Comment:
This should generate a test for the centralized web server once it is merged in, right?

Reply:
This only works for me if you add the channel feature as well, like this: 
`list_tests_json:
  cargo test --verbose --profile=release-lto --features=full-ci,channel-async-std --lib --bins --tests --benches --package=hotshot-testing --no-fail-fast -- --test-threads=1 --nocapture -Zunstable-options --format json`